Mathlib/Data/Nat/Cast/Order/Ring.lean

```diff
@@ -82,22 +82,27 @@ theorem cast_tsub [CommSemiring α] [PartialOrder α] [IsOrderedRing α] [Canoni
   · rcases le_iff_exists_add'.mp h with ⟨m, rfl⟩
     rw [add_tsub_cancel_right, cast_add, add_tsub_cancel_right]
 
-section LinearOrderedRing
-variable [Ring R] [LinearOrder R] [IsStrictOrderedRing R] {m n : ℕ} {m n : ℕ}
```

Comment:
This doubled `{m n : ℕ} {m n : ℕ}` looks wrong. I'm happy you're getting rid of it!